### PR TITLE
Traffic policy guide

### PR DESCRIPTION
**Pull request by ned1313**
Adding a guide on creating a basic Traffic Policy resource for the Pet Store application.

### CODE REVIEW

File: docs/content/guides/traffic_policy.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+  - retries:
+      attempts: 5
+      perTryTimeout: 0.005s
+    route:
+    - destination:
+        host: petstore.default.svc.cluster.local
+    timeout: 0.100s
+```
+
+As we can see above, the proper retry and timeout settings have been applied to the VirtualService from the Service Mesh Hub TrafficPolicy. This feature can be extended to configure many services across multiple service meshes and clusters.
```
Comment (joekelley) [inline docs/content/guides/traffic_policy.md:181]: let's also add a note that many other features can be configured via the traffic policy (e.g. fault injection, traffic mirroring) and link users to the API https://docs.solo.io/service-mesh-hub/latest/reference/api/traffic_policy/#networking.smh.solo.io.TrafficPolicySpec

Comment (ned1313) [reply docs/content/guides/traffic_policy.md]: Added.

**Pull request merged**